src/main/app/common/court-calculations/courtDetermination.ts

```diff
@@ -0,0 +1,26 @@
+import { Moment } from 'moment'
+
+export enum DecisionType {
+  CLAIMANT = 'CLAIMANT',
+  DEFENDANT = 'DEFENDANT',
+  COURT = 'COURT'
+}
+
+export class CourtDetermination {
+
+  static calculateDecision (defendantPaymentDate: Moment,
+                             claimantPaymentDate: Moment,
+                             courtGeneratedPaymentDate: Moment): DecisionType {
+
+    if (claimantPaymentDate.isAfter(defendantPaymentDate) || claimantPaymentDate.isAfter(courtGeneratedPaymentDate)) {
```

Comment:
could you please try isSameOrAfter instead of isAfter as dates can be same and run all tests